### PR TITLE
Update tiflash supported pushdown expression list

### PR DESCRIPTION
**Pull request by zanmato1984**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/use-tiflash.md [hunk 1]
```diff
@@ -234,9 +234,15 @@ TiFlash 支持部分算子的下推，支持的算子如下：
...
-```
+数学函数: +, -, /, *, >=, <=, =, !=, <, >, round, abs
+逻辑函数: and, or, not, case when, if, ifnull, isnull, in
+位运算: bitand, bitor, bigneg, bitxor
+字符串函数: substr, replace, concat, concat_ws, left, right
+日期函数: date_format, timestampdiff, from_unixtime, unix_timestamp, str_to_date, date_add, date_sub, datediff, year, month, day, extract
+JSON 函数: json_length
+转换函数: cast(int as double), cast(int as decimal), cast(int as string), cast(int as time), cast(double as int), cast(double as decimal), cast(double as string), cast(double as time), cast(string as int), cast(string as double), cast(string as decimal), cast(string as time), cast(decimal as int), cast(decimal as string), cast(decimal as time), cast(time as int), cast(time as decimal), cast(time as string)
+聚合函数: min, max, sum, count, avg, approx_count_distinct
+`
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:245]:
```suggestion
```
```

File: tiflash/use-tiflash.md [hunk 2]
```diff
@@ -234,8 +234,14 @@ TiFlash 支持部分算子的下推，支持的算子如下：
...
-+, -, /, *, >=, <=, =, !=, <, >, ifnull, isnull, bitor, in, bitand, or, and, like, not, case when, month, substr, timestampdiff, date_format, from_unixtime, json_length, if, bitneg, bitxor,
-round without fraction, cast(int as decimal), date_add(datetime, int), date_add(datetime, string), min, max, sum, count, avg, approx_count_distinct
+数学函数: +, -, /, *, >=, <=, =, !=, <, >, round, abs
+逻辑函数: and, or, not, case when, if, ifnull, isnull, in
+位运算: bitand, bitor, bigneg, bitxor
+字符串函数: substr, replace, concat, concat_ws, left, right
+日期函数: date_format, timestampdiff, from_unixtime, unix_timestamp, str_to_date, date_add, date_sub, datediff, year, month, day, extract
+JSON 函数: json_length
+转换函数: cast(int as double), cast(int as decimal), cast(int as string), cast(int as time), cast(double as int), cast(double as decimal), cast(double as string), cast(double as time), cast(string as int), cast(string as double), cast(string as decimal), cast(string as time), cast(decimal as int), cast(decimal as string), cast(decimal as time), cast(time as int), cast(time as decimal), cast(time as string)
+聚合函数: min, max, sum, count, avg, approx_count_distinct
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:244]:
```suggestion
数学函数：+, -, /, *, >=, <=, =, !=, <, >, round, abs

逻辑函数：and, or, not, case when, if, ifnull, isnull, in

位运算：bitand, bitor, bigneg, bitxor

字符串函数：substr, replace, concat, concat_ws, left, right

日期函数：date_format, timestampdiff, from_unixtime, unix_timestamp, str_to_date, date_add, date_sub, datediff, year, month, day, extract

JSON 函数：json_length

转换函数：cast(int as double), cast(int as decimal), cast(int as string), cast(int as time), cast(double as int), cast(double as decimal), cast(double as string), cast(double as time), cast(string as int), cast(string as double), cast(string as decimal), cast(string as time), cast(decimal as int), cast(decimal as string), cast(decimal as time), cast(time as int), cast(time as decimal), cast(time as string)

聚合函数：min, max, sum, count, avg, approx_count_distinct
```

**Comment by zanmato1984** [comment]
@TomShawn PTAL

**Review by windtalker — APPROVED** [review]
LGTM

File: tiflash/use-tiflash.md [hunk 3]
```diff
@@ -233,10 +233,14 @@ TiFlash 支持部分算子的下推，支持的算子如下：
 
 在 TiDB 中，算子之间会呈现树型组织结构。一个算子能下推到 TiFlash 的前提条件，是该算子的所有子算子都能下推到 TiFlash。因为大部分算子都包含有表达式计算，当且仅当一个算子所包含的所有表达式均支持下推到 TiFlash 时，该算子才有可能下推给 TiFlash。目前 TiFlash 支持下推的表达式包括：
 
-```
-+, -, /, *, >=, <=, =, !=, <, >, ifnull, isnull, bitor, in, bitand, or, and, like, not, case when, month, substr, timestampdiff, date_format, from_unixtime, json_length, if, bitneg, bitxor,
-round without fraction, cast(int as decimal), date_add(datetime, int), date_add(datetime, string), min, max, sum, count, avg, approx_count_distinct
-```
+* 数学函数: `+, -, /, *, >=, <=, =, !=, <, >, round(int), round(double), abs, floor(int), ceil(int), ceiling(int)`
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:236]:
```suggestion
* 数学函数：`+, -, /, *, >=, <=, =, !=, <, >, round(int), round(double), abs, floor(int), ceil(int), ceiling(int)`
```

File: tiflash/use-tiflash.md [hunk 4]
```diff
@@ -233,10 +233,14 @@ TiFlash 支持部分算子的下推，支持的算子如下：
...
+* 逻辑函数: `and, or, not, case when, if, ifnull, isnull, in`
+* 位运算: `bitand, bitor, bigneg, bitxor`
+* 字符串函数: `substr, char_length, replace, concat, concat_ws, left, right`
+* 日期函数: `date_format, timestampdiff, from_unixtime, unix_timestamp(int), unix_timestamp(decimal), str_to_date(date), str_to_date(datetime), date_add(string, int), date_add(datetime, int), date_sub(datetime, int), date_sub(string, int), datediff, year, month, day, extract(datetime)`
+* JSON 函数: `json_length`
+* 转换函数: `cast(int as double), cast(int as decimal), cast(int as string), cast(int as time), cast(double as int), cast(double as decimal), cast(double as string), cast(double as time), cast(string as int), cast(string as double), cast(string as decimal), cast(string as time), cast(decimal as int), cast(decimal as string), cast(decimal as time), cast(time as int), cast(time as decimal), cast(time as string)`
+* 聚合函数: `min, max, sum, count, avg, approx_count_distinct`
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:243]:
```suggestion
* 逻辑函数：`and, or, not, case when, if, ifnull, isnull, in`
* 位运算：`bitand, bitor, bigneg, bitxor`
* 字符串函数：`substr, char_length, replace, concat, concat_ws, left, right`
* 日期函数：`date_format, timestampdiff, from_unixtime, unix_timestamp(int), unix_timestamp(decimal), str_to_date(date), str_to_date(datetime), date_add(string, int), date_add(datetime, int), date_sub(datetime, int), date_sub(string, int), datediff, year, month, day, extract(datetime)`
* JSON 函数：`json_length`
* 转换函数：`cast(int as double), cast(int as decimal), cast(int as string), cast(int as time), cast(double as int), cast(double as decimal), cast(double as string), cast(double as time), cast(string as int), cast(string as double), cast(string as decimal), cast(string as time), cast(decimal as int), cast(decimal as string), cast(decimal as time), cast(time as int), cast(time as decimal), cast(time as string)`
* 聚合函数：`min, max, sum, count, avg, approx_count_distinct`
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**